test/asset/raw_datasets.jsonl

```diff
@@ -30,9 +30,6 @@
 {"dataset_name": "IWSLT2017", "split": "train", "NUM_LINES": 206112, "MD5": "aca701032b1c4411afc4d9fa367796ba", "URL": "https://drive.google.com/u/0/uc?id=12ycYSzLIG253AFN35Y6qoyf9wtkOjakp", "first_line": "c75166d2ffde3978586af8a8ebdf6450"}
 {"dataset_name": "IWSLT2017", "split": "valid", "NUM_LINES": 888, "MD5": "aca701032b1c4411afc4d9fa367796ba", "URL": "https://drive.google.com/u/0/uc?id=12ycYSzLIG253AFN35Y6qoyf9wtkOjakp", "first_line": "c43021713268b2efc08a255c191f2c74"}
 {"dataset_name": "IWSLT2017", "split": "test", "NUM_LINES": 1568, "MD5": "aca701032b1c4411afc4d9fa367796ba", "URL": "https://drive.google.com/u/0/uc?id=12ycYSzLIG253AFN35Y6qoyf9wtkOjakp", "first_line": "cfff6f23c564bc4cb372bee4987f6707"}
-{"dataset_name": "WMT14", "split": "train", "NUM_LINES": 4500966, "MD5": "874ab6bbfe9c21ec987ed1b9347f95ec", "URL": "https://drive.google.com/uc?export=download&id=0B_bZck-ksdkpM25jRUN2X2UxMm8", "first_line": "27a5871c90db257250806f52ba6aff0c"}
-{"dataset_name": "WMT14", "split": "valid", "NUM_LINES": 3000, "MD5": "874ab6bbfe9c21ec987ed1b9347f95ec", "URL": "https://drive.google.com/uc?export=download&id=0B_bZck-ksdkpM25jRUN2X2UxMm8", "first_line": "a302bd241b4c3138a400e38a846f93b0"}
-{"dataset_name": "WMT14", "split": "test", "NUM_LINES": 3003, "MD5": "874ab6bbfe9c21ec987ed1b9347f95ec", "URL": "https://drive.google.com/uc?export=download&id=0B_bZck-ksdkpM25jRUN2X2UxMm8", "first_line": "140245f6a92f95225150f717e2d7a1a7"}
```

Comment:
We probably want to keep around raw experimental datasets and the corresponding testing.